README.md

```diff
@@ -39,10 +50,17 @@ Version 1.2.0 of the plugin introduced improved logging for debugging purposes.
 
 You can also try chatting with us in the #gitlab-plugin channel on the Freenode IRC network.
 
-# Supported GitLab versions
+# Plugin Modes and Supported GitLab versions
+
+The Plugin supports two operation modes: **Modern** and **Legacy**. The main difference between these modes is how they deal with Merge Requests from GitLab. While the Legacy mode requires setting up multiple Git repositories and using parameterized builds, the Modern mode works with a single Git repository and relies on GitLab's availability of Merge Requests from the `origin` repository. The rest of this document is split for these two modes of operation (unless specified).
 
-* 7.14.x where it emulates Jenkins as a GitLabCI Web Service
-* 8.1.x and newer via the new commit status API that supports with external CI services like Jenkins
+The following versions of GitLab are supported by each of the modes:
+
+* Modern
+  * 8.1.x and newer
```

Comment:
Why is version **8.1.x** listed in both **Modern** and **Legacy**? The modes should be mutually exclusive, shouldn't they?